src/allmydata/immutable/downloader/share.py

```diff
@@ -475,7 +475,7 @@ def _signal_corruption(self, f, start, offset):
         # there was corruption somewhere in the given range
         reason = "corruption in share[%d-%d): %s" % (start, start+offset,
                                                      str(f.value))
-        self._rref.callRemote(
+        return self._rref.callRemote(
```

Comment:
I couldn't find any callers of `_signal_corruption` that do anything with the return value.  This seems like it won't make any difference?